pages/documentation/guidance/performance/research.md

```diff
@@ -53,7 +53,7 @@ This process helped shine light on the pros and cons of each tool, which allowed
 
 #### User interviews
 
-The user interviews took place over the course of three weeks in March 2017. We began by interviewing a member of the 18F team who has expertise in HTTPS and HTTP/2 and shared with us the complementary ways in which it can help to improve web performance. Next, we interviewed a second member of the 18F team who manages [Pulse](https://pulse.cio.gov/), which publishes how federal agencies are meeting best practices on the web with respect to HTTPS and the [Digital Analytics Program](https://www.digitalgov.gov/services/dap/).
+The user interviews took place over the course of three weeks in March 2017. We began by interviewing a member of the 18F team who has expertise in HTTPS and HTTP/2 and shared with us the complementary ways in which it can help to improve web performance. Next, we interviewed a second member of the 18F team who manages [Pulse](https://pulse.cio.gov/), which publishes how federal agencies are meeting best practices on the web with respect to HTTPS and the [Digital Analytics Program](https://digital.gov/guides/dap/).git
```

Comment:
@amyleadem @mahoneycm - This link needs to be double-checked as it is part of two new links we are receiving errors on

Reply:
> [!note]
> Confirming this points to digital.gov's DAP guide

Reply:
**issue:** Looks like there was an accidental addition of `git` in the text here

Reply:
Went ahead and removed this in [d64ad66](https://github.com/uswds/uswds-site/pull/2995/commits/d64ad6603105ff5d7df7af5f32a2c1f193f9abd6)